pkg/util/util.go

```diff
@@ -168,7 +168,7 @@ var letterRunes = []rune("abcdefghijklmnopqrstuvwxyz")
 func RandStringRunes(n int) string {
 	b := make([]rune, n)
 	for i := range b {
-		b[i] = letterRunes[rand.Intn(len(letterRunes))]
+		b[i] = letterRunes[rand.Intn(len(letterRunes))] // #nosec G404 -- used only in tests
```

Comment:
This file lives in the  pkg/util/util.go path. This does not seem like a test-only package.
If it is test-only, it should be moved probably.
I think we should move / switch to RandStringBytes before we add this exception, because it might be used in non-tests in the future.

Reply:
@wallrj

Reply:
I agree that it should be changed, but not here. I'll do that in a separate PR. See earlier comment and linked issue.